content/en/samples/posts/2024-12-03-stand-by-app-webos.md

```diff
@@ -95,29 +101,49 @@ Clicking on a widget opens a modal where users can add schedules in the calendar
 
 **When deleting a schedule** -> The schedule is deleted via an API, and the cached data is also updated.
 
+<img src="https://github.com/KNU-LG/frontend/raw/main/src/assets/docsImage/image-7.png">
+
 ### Dimming Feature
 
 This is managed through a dimming provider. Using `window.addEventListener`, it detects `mousemove`, `keydown`, `touchstart`, and `click` events. If no such events are detected for 10 seconds, an additional layer is applied to darken the screen.
 
+<img src="https://github.com/KNU-LG/frontend/raw/main/src/assets/docsImage/image-8.png">
+
 ### Dark / Light Mode
 
 The `Context API` is used to globally manage and toggle between dark and light modes. Each mode's CSS values are globally managed to affect the UI.
 
-<br/><br>
+- Light Mode
+<img src="https://github.com/KNU-LG/frontend/raw/main/src/assets/docsImage/image-9.png">
 
-# Usage Technology Stack
+- Dark Mode
+<img src="https://github.com/KNU-LG/frontend/raw/main/src/assets/docsImage/image-10.png">
 
-> System Structure Chart
+<br/><br>
 
-<img src="https://github.com/user-attachments/assets/70ae41d0-5da0-42ba-b49e-a24afee254a1">
+# Usage Technology Stack
 
-> Database ERD
+It is a diagram of the overall structure of our system.
+<p align = "center">
```

Comment:
need to change this block to `{{<figure >}}` format.

example)
{{< figure src="url_of_image" caption="caption of image" alt="alternative name of imange" >}}